java/client/src/org/openqa/selenium/Platform.java

```diff
@@ -72,6 +72,13 @@ public Platform family() {
     }
   },
 
+  WIN10("windows 10", "win10") {
```

Comment:
we'll also have to think about adding "Windows Server 2014" or whatever they come up with later.